src/index.ts

```diff
@@ -17,6 +24,45 @@ import * as https from 'https'
 import * as fs from 'fs'
 import protectOathkeeper from './middleware/oathkeeper'
 import protectSimple from './middleware/simple'
+import bodyParser from 'body-parser'
+import csrf from 'csurf'
+import session from 'express-session'
+
+const csrfProtection = csrf({ cookie: true })
+
+const protectOathKeeper = jwt({
```

Comment:
Given that we don't really use this set up any more - should we just remove it? Alternatively, this is handled in: https://github.com/ory/kratos-selfservice-ui-node/blob/master/src/middleware/oathkeeper.ts

Reply:
Also correct, I missed this duplicate.  Done.